zkevm-circuits/src/evm_circuit/execution/logs.rs

```diff
@@ -126,8 +126,11 @@ impl<F: Field> ExecutionGadget<F> for LogGadget<F> {
         );
 
         let copy_rwc_inc = cb.query_cell();
-        let dst_addr = (1u64 << 32).expr() * TxLogFieldTag::Data.expr()
-            + (1u64 << 48).expr() * (cb.curr.state.log_id.expr() + 1.expr());
+        let dst_addr = build_tx_log_expression(
+            0.expr(),
+            TxLogFieldTag::Data.expr(),
+            cb.curr.state.log_id.expr(),
```

Comment:
```suggestion
            cb.curr.state.log_id.expr() + 1.expr(),
```

Reply:
I was conviced I did it :exploding_head:

Reply:
I did it but didn't commit it